File: python-example/model/customer.py
class Customer:
    def __init__(self, firstname=None, lastname=None, address=None, postcode=None, city=None, country=None, zone=None,
                 email=None, phone=None, password=None):
        self.address = address
        self.postcode = postcode
        self.city = city
        self.country = country
        self.zone = zone
        self.email = email
        self.phone = phone
        self.password = password
        self.firstname = firstname
        self.lastname = lastname

File: ruby-example/model/customer.rb
class Customer
  attr_reader :address
  attr_reader :postcode
  attr_reader :city
  attr_reader :country
  attr_reader :zone
  attr_reader :email
  attr_reader :phone
  attr_reader :password
  attr_reader :firstname
  attr_reader :lastname

  def initialize(firstname: nil, lastname: nil, address: nil, postcode: nil, city: nil, country: nil, zone: nil, email: nil, phone: nil, password: nil)
    @address = address
    @postcode = postcode
    @city = city
    @country = country
    @zone = zone
    @email = email
    @phone = phone
    @password = password
    @firstname = firstname
    @lastname = lastname
  end
end

